src/Backend/OnboardingInsuranceAPI/Services/DataContext.cs

```diff
@@ -7,19 +9,25 @@ public class DataContext : DbContext
     public DataContext(DbContextOptions options) : base(options)
     {
     }
-
+    public DbSet<CarInsuranceInfo> Insurances { get; set; }
     public DbSet<UserInfo> Users { get; set; }
 
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
         modelBuilder.Entity<UserInfo>()
-            .HasPartitionKey(o => o.Pid)
-            .HasData(new UserInfo
+        .Property(e => e.Birthdate)
+        .HasConversion(v => v.Value.ToString("yyyy-MM-dd"), v => DateOnly.Parse(v));
+
+        modelBuilder.Entity<UserInfo>()
+        .ToContainer("Users")
```

Comment:
These lines should be one tab to the right as they not on the same level as the "modelBuilder"